.github/workflows/benchmarks.yml

```diff
@@ -4,13 +4,6 @@ on:
   push:
     branches:
       - main
-    paths:
```

Comment:
you should include 
```
- "**.py"
- ".github/workflows/**.yml"
```

in the workflow so that if any changes are made in these file the action will be triggered.

Reply:
I do not think we should put it. In that case, if I do a new MR where I change the benchmarks.yml a new push with new benchmarks will be done. Or at least is what I understand.

Reply:
`pull_request: `--> this is for PR
`push:` -->  is for merge

you have to add above paths to `push:`

if it makes sense ?

Reply:
Aaaaaaaah, okey. The action is triggered when a "merge" is made to "main" and it changes one of the following "paths". If what I said is correct, it makes sense.

Reply:
yess... 😃
no problem